app/queries/vacancy_filter_query.rb

```diff
@@ -33,6 +33,24 @@ def call(filters) # rubocop:disable Metrics/AbcSize
 
   private
 
+  def add_organisation_type_filters(filters, built_scope)
+    return built_scope unless filters[:organisation_types].present?
+
+    selected_school_types = []
+
+    if filters[:organisation_types].include?("Academy")
+      selected_school_types.push("Academy", "Academies", "Free schools", "Free school")
+    end
+
+    if filters[:organisation_types].include?("Local authority maintained schools")
+      selected_school_types << "Local authority maintained schools"
+    end
+
```

Comment:
I have used this subquery to avoid using distinct here which fixes the error we are getting, although I think it will make the query slightly more expensive.